ci/Jenkinsfile.linux

```diff
@@ -39,7 +39,6 @@ pipeline {
     NPM_CONFIG_CACHE = '/var/tmp/npm'
     LEIN_HOME = '/var/tmp/lein'
     QT_PATH = '/opt/qt'
-    STATUSIM_APPIMAGE = '/opt/StatusImAppImage.zip'
```

Comment:
What makes more sense is instead of doing:
```
-v /opt/StatusImAppImage_20181113.zip:/opt/StatusImAppImage.zip:ro
```
We should do:
```
-v /opt/desktop-files:/opt/desktop-files:rw"
```
And:
```
STATUSIM_APPIMAGE_DIR = '/opt/desktop-files'
```
And let it download it.

Reply:
@jakubgs thanks!! will try